### PR TITLE
Docs, Hollow Knight: Update en_Hollow Knight.md

### PR DESCRIPTION
**Pull request by AutomaticFrenzy**
Add logic requirements

## What is this fixing or adding?

Helping guide new players to benchwarping to the start being in logic

## How was this tested?

Wasn't

## If this makes graphical changes, please attach screenshots.

Adds text to webpage for hollow knight

### CODE REVIEW

**Comment by Berserker66** [comment]
@dewiniaid

**Review by agilbert1412 — APPROVED** [review]

File: worlds/hk/docs/en_Hollow Knight.md
Comment (mDuo13) [inline worlds/hk/docs/en_Hollow Knight.md:26]:
```suggestion
Randomization may expect you to use the Benchwarp mod to return to the starting location. Warping elsewhere will never be required to complete the game. For more details, see the documentation for the Hollow Knight randomizer mod, below:

```

**Review by mDuo13 — COMMENTED** [review]
This is good, but I suggested a way to rephrase it to be even better.

**Comment by Exempt-Medic** [comment]
I don't think linking to the base randomizer readmes is a great idea. They can change over time and the AP implementation is quite different from the base one, especially as it does not include several of the settings.

File: worlds/hk/docs/en_Hollow Knight.md
Comment (BadMagic100) [inline worlds/hk/docs/en_Hollow Knight.md:26]: This almost makes it sound like you're expected not to benchwarp anywhere other than start which isn't really true - maybe a ", though warping to other benches you have been to already is expected for convenience" or something along those lines

File: worlds/hk/docs/en_Hollow Knight.md
Comment (BadMagic100) [inline worlds/hk/docs/en_Hollow Knight.md:28]: While these are good resources, I'd prefer if we made more clear that these are resources for the standalone randomizer that may not be completely accurate to Archipelago.

**Comment by ThePhar** [comment]
@AutomaticFrenzy have you had a chance to look over the review from BadMagic?

**Comment by AutomaticFrenzy** [comment]
Open to however people want to phrase it, I just want it to mention bench warping or give a link to a document that says that. Beyond that I don't have any input

**Comment by black-sliver** [comment]
Well, you are supposed to either rework the PR to meet what the world maintainer's requirements, or argument why it should be merged as it is. If you are not interested in working on this, please close the PR.

**Pull request closed without merging**
